plugins/readtimer.koplugin/main.lua

```diff
@@ -80,17 +80,19 @@ function ReadTimer:addToMainMenu(menu_items)
         end,
         sub_item_table = {
             {
-                text = _("Time"),
+                text = _("Point in Time"),
                 keep_menu_open = true,
                 callback = function(touchmenu_instance)
                     local now_t = os.date("*t")
                     local curr_hour = now_t.hour
                     local curr_min = now_t.min
-                    local time_widget = TimeWidget:new{
+                    local time_widget = DateTimeWidget:new{
+                        is_date = false,
                         hour = curr_hour,
                         min = curr_min,
                         ok_text = _("Set timer"),
                         title_text =  _("Set reader timer"),
+                        info_text = _("Enter time in hours an minutes."),
```

Comment:
```suggestion
                        title_text =  _("New alarm"),
```

Reply:
Is it really clear to any user that something like
   
![grafik](https://user-images.githubusercontent.com/36999612/134500571-7c6a93d3-4529-4e83-a469-9646307f1fe4.png)

shall be 00:00 (0 h 0 min)?

Or wouldn't a info text be more user friendly, something like

![grafik](https://user-images.githubusercontent.com/36999612/134500863-dbbd9729-ced2-42ea-aab8-1e1bc56106fa.png)